analyzer/build.gradle.kts

```diff
@@ -53,7 +53,6 @@ repositories {
 dependencies {
     api(project(":model"))
 
-    implementation(project(":clients:clearly-defined"))
```

Comment:
I believe the dependency on the SW360 client (and the corresponding repository declaration) should be removed, too. And maybe the commit message could briefly state that these dependencies that been moved to dedicated projects in the commit before. Or even squash this commit into the previous one.

Reply:
I have squashed it.